PsychSourceGL/Source/Common/Base/PythonGlue/PsychScriptingGluePython.c

```diff
@@ -50,18 +50,16 @@
 
 #if PSYCH_LANGUAGE == PSYCH_PYTHON
 
-// Import NumPy array handling functions: Require at least NumPy v 1.7, released
-// in February 2013:
-#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+// Import NumPy array handling functions: Require at least NumPy v 1.13.0, released
+// in June 2017:
+#define NPY_NO_DEPRECATED_API NPY_1_13_API_VERSION
 #include <numpy/arrayobject.h>
-// Define after include, to avoid compiler warning and pass runtime loader compat check:
-#undef NPY_FEATURE_VERSION
-#define NPY_FEATURE_VERSION NPY_1_7_API_VERSION
-
-// Can not use NPY_TITLE_KEY macro if compat limited api is selected.
-// However, i have no clue what we'd use it for, so there...
-#ifdef Py_LIMITED_API
-#undef NPY_TITLE_KEY
+
+#if defined(Py_LIMITED_API) && defined(__GNUC__) && (__GNUC__ < 10)
+#error \
+"This version of gcc has a bug and cannot compile this code. \
+Please upgrade gcc to a version >= 10.1 and try again. \
```

Comment:
Should mention one can also just build without the limited api, instead of compiler upgrade.